pkgs/applications/misc/obsidian/default.nix

```diff
@@ -34,23 +34,23 @@ in stdenv.mkDerivation rec {
 
   src = fetchurl {
     url =
-      "https://github.com/obsidianmd/obsidian-releases/releases/download/v${version}/obsidian-${version}.asar.gz";
-    sha256 = "AkPx7X00kEds7B1syXJPSV1+TJlqQ7NnR6w9wSG2BRw=";
+      "https://github.com/obsidianmd/obsidian-releases/releases/download/v${version}/obsidian-${version}.tar.gz";
+    sha256 = "1acy0dny04c8rdxqvsq70aa7vd8rgyjarcrf57mhh26ai5wiw81s";
   };
 
   nativeBuildInputs = [ makeWrapper graphicsmagick ];
 
   unpackPhase = ''
```

Comment:
Is this `unpackPhase` still necessary then? Shouldn't the default builder take care of this?

Reply:
to be honest, I don't know. I'll try removing and see if it works.

Reply:
I tried removing `unpackPhase` completely and this is the error which showed up:
```
install: missing destination file operand after '/nix/store/bqspcdjzh74rx1dg1rviqm8aa9gnyyi9-obsidian-0.11.0/share/electron/'
```
am I missing something here? 🤔

Reply:
turns out it was caused by the glob / wildcard -- I was pushing a dirty state and now it works